src/modules/design/bpmn-diff-view/bpmn-diff-view.html

```diff
@@ -18,7 +22,12 @@
         <span class="diff-legend-entry"><i class="fas fa-stop orange"></i> Changed</span>
         <span class="diff-legend-entry diff-legend-entry-last"><i class="fas fa-stop purple"></i> Layout Changed</span>
       </div>
-      <span class="diff-canvas-title">${diffModeTitle}</span>
+      <span if.bind="diffModeIsNewVsOld" class="diff-canvas-title">
+        <p class="diffMode">${currentXmlIdentifier} vs. ${previousXmlIdentifier}</p>
+      </span>
+      <span else class="diff-canvas-title">
+        <p class="diffMode">${previousXmlIdentifier} vs. ${currentXmlIdentifier}</p>
```

Comment:
css-classes should be snake-cased.
```suggestion
        <p class="diff-mode">${previousXmlIdentifier} vs. ${currentXmlIdentifier}</p>
```